models/penalty_charge_notice.rb

```diff
@@ -21,6 +21,10 @@ def discounted?
     ((14.days.ago)..(Time.now)).include?(issued_at)
   end
 
+  def description
+    charge_type.humanize + (discounted? ? ' (early payment discount)' : '')
```

Comment:
I'm not sure we need to include the discount on the GOV.UK payment page – showing it on our own page would probably be enough?